packages/account/src/providers/transaction-request/transaction-request.test.ts

```diff
@@ -113,6 +115,40 @@ describe('TransactionRequest', () => {
     expect(transactionRequest.witnesses.length).toEqual(2);
     expect(transactionRequest.witnesses).toStrictEqual([mockSignedTx, mockSignedTx]);
   });
+
+  it.only('adds account based witnesses', async () => {
```

Comment:
I think this went unnoticed.

```suggestion
  it('adds account based witnesses', async () => {
```

Reply:
Oops, I thought this was ready for review.

Sorry for the noise.

Reply:
No problem, you caught it before review 🕵🏻